feature-libs/product/configurators/common/src/shared/testing/configuration-test-data.ts

```diff
@@ -0,0 +1,164 @@
+import { Configurator, GenericConfigurator } from '@spartacus/core';
+
+export const PRODUCT_CODE = 'CONF_LAPTOP';
+export const CONFIG_ID = '1234-56-7890';
+
+export const GROUP_ID_1 = '1234-56-7891';
+export const GROUP_ID_2 = '1234-56-7892';
+export const GROUP_ID_3 = '1234-56-7893';
+export const GROUP_ID_4 = '1234-56-7894';
+export const GROUP_ID_5 = '1234-56-7895';
+export const GROUP_ID_6 = '1234-56-7896';
+export const GROUP_ID_7 = '1234-56-7897';
+export const GROUP_ID_8 = '1234-56-7898';
+export const GROUP_ID_9 = '1234-56-7899';
+export const GROUP_ID_10 = '1234-56-7900';
+
+const CONFIGURATOR_URL =
```

Comment:
I guess this test data is being re-used between multiple tests?
Is it exposed from the library?
Is all the data re-used between the tests?

Reply:
Yes...